rpcs3/Crypto/unself.cpp

```diff
@@ -1109,8 +1104,11 @@ bool SELFDecrypter::DecryptNPDRM(u8 *metadata, u32 metadata_size)
 	}
 	else if (ctrl->npdrm.license == 3)  // Free license.
 	{
-		// Use the NP_KLIC_FREE.
-		memcpy(npdrm_key, NP_KLIC_FREE, 0x10);
+		// Use klicensee if available.
```

Comment:
Its the only spot where it seems to actually be used. Theres cases where the edat has local license flag, and npDrmIsAvailable() is called with a klicense_key, but in that case it needs to be ignored and the rap is used instead. All the cases I've found that actually use the klicense given are flagged with 'free-license'